### PR TITLE
Revert "Update Cassandra chart version to latest"

### PR DESCRIPTION
**Pull request by vkamra**
Reverts kanisterio/kanister#755. This is breaking in the pipeline

### CODE REVIEW

**Review by ewhamilton — APPROVED** [review]

File: pkg/app/cassandra.go [hunk 1]
```diff
@@ -47,17 +47,17 @@ type CassandraInstance struct {
...
 			Release:  appendRandString(name),
-			RepoURL:  helm.BitnamiRepoURL,
-			RepoName: helm.BitnamiRepoName,
+			RepoURL:  helm.IncubatorRepoURL,
 			Chart:    "cassandra",
+			RepoName: helm.IncubatorRepoName,
+			Version:  "0.13.3",
 			Values: map[string]string{
-				"image.repository":    "kanisterio/cassandra",
+				"image.repo":    "kanisterio/cassandra",
```
Comment (ewhamilton) [inline pkg/app/cassandra.go:60]: Does this need to be `image.repository`?

Comment (ewhamilton) [reply pkg/app/cassandra.go:60]: And further checking of #755 shows that image.repo is the correct revert. Removing hold-off label.

Comment (ewhamilton) [reply pkg/app/cassandra.go:60]: But... this file fail travis because it needs a gofmt.

**Comment by ewhamilton** [comment]
Travis failing for gofmt on pkg/app/cassandra.go and mongo test.

File: pkg/app/cassandra.go (hunk 1 above)
Comment (viveksinghggits) [reply pkg/app/cassandra.go]: I will ask @arush-sal to take a look into this.

**Pull request merged**